app/services/ClickHandler.coffee

```diff
@@ -0,0 +1,66 @@
+$ = require 'jquery'
+
+class ClickHandler
+
+  constructor: (@board, two, @clicked = []) ->
+    return unless @board.cells?
+    for row in @board.cells
+      break if row.length is 0
+      for cell in row
+        (@addToClicked cell) if cell.isSelected
+
+  bindDefaultClick: (board) ->
+    $('body').click (e) =>
+      e.preventDefault()
+      @resetClicked()
+
+  bindClickTo: (cells) ->
+    if cells.bindClick?
+      cells.bindClick() 
+      return
+    for row in cells
+      if row.bindClick?
+        row.bindClick()
+        return
```

Comment:
@s-gogna, yes.  Thats the idea.  You can either pass in an object, an array or a 2d array and it will bind each element in the appropriate collection.

@bmwwoo Thats not working for me for some reason